src/main/java/toast/specialMobs/TickHandler.java

```diff
@@ -36,6 +38,18 @@ public void onServerTick(TickEvent.ServerTickEvent event) {
                     }
                     entry.replace();
                 }
+                // If processing the entity queue would take longer than a minute, drop the list to avoid running out of
+                // memory
+                if (TickHandler.entityStack.size() > 10 * 20 * 60) {
+                    long now = System.currentTimeMillis();
+                    if (Math.abs(now - lastStackTrimmedMessageTime) < 10_000) {
```

Comment:
Add ` || lastStackTrimmedMessageTime == 0` to condition; otherwise logging will never happen.

Reply:
Oh I accidentally inverted the condition, it should be `>`